pombola/kenya/static/js/survey.js

```diff
@@ -0,0 +1,75 @@
+(function() {
+
+    // Percentage of users to display the survey for
+    var p = 0.6;
+
+    // Is cookie existing?
+
+    var ct = null;
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') {
+            c = c.substring(1,c.length);
+        }
+        if (c.indexOf('ms_srv_t=') == 0) {
+            // Cookie for time...
+            var ct = c.substring('ms_srv_t='.length,c.length);
+        }
+        if (c.indexOf('ms_srv_r=') == 0) {
+            // Cookie for referrer...
+            var cr = c.substring('ms_srv_r='.length,c.length);
+        }
+    }
+
+    if (ct == null) {
+        // No cookie!
+        if (Math.random() < p) {
+            // Chosen to survey
+            // Set the cookie to current timestamp
+            var t = Math.round(new Date().getTime() / 1000);
+            document.cookie = 'ms_srv_t='+t+'; path=/';
+            document.cookie = 'ms_srv_r='+document.referrer+'; path=/';
+            ct = t;
+            cr = document.referrer;
+
+        } else {
+            // Not chosen to survey
+            // Set cookie to X
+            document.cookie = 'ms_srv_t=X; path=/';
+            ct = 'X'
+        }
+    }
+
+    // Only bother to do this if the cookie is set and the link exists
+    if (ct != 'X' && !! document.getElementById('ms_srv_wrapper')) {
```

Comment:
The `!!` isn't needed here, I think - I don't see why this needs to be a boolean type as well as just truthy?